sphinx/texinputs/sphinxhowto.cls

```diff
@@ -92,7 +92,7 @@
 % so no page break before it.
 \newenvironment{sphinxthebibliography}[1]{%
   \begin{thebibliography}{1}%
-  \addcontentsline{toc}{section}{\bibname}}{\end{thebibliography}}
+  \addcontentsline{toc}{section}{\ifx\bibname\bibname\else\refname\fi}}{\end{thebibliography}}
```

Comment:
this is no good `\ifx\bibname\bibname` test will always be true, so the whole `\ifx..\fi` expands to nothing... the `\bibname` issue is a bit complicated (Japanese documents use `jreport.cls` hence need `\bibname`, others use `article.cls` which has only `\refname`, `\bibname`  exists due to `babel` but is not used by `thebibliography` and should not be used).